### DISCUSSION TITLE
MTest + Matlab

### DISCUSSION THREAD

**Discussion by fabiomachadofonseca** [thelfer/tfel#416] [Q&A]
Hello @thelfer,

I am currently working with MFront/MTest, and I would like to integrate it with Matlab to optimize some constants. Have you ever done this before?

Thank you for your assistance.
Fabio

**Comment by thelfer** [comment]
Dear @fabiomachadofonseca,

Thanks for your interest. I never did this.  I remember someone telling me that it was feasible, but can't remember who.

Let me know if you have some success in the matter.

Regards,
Thomas

**Reply by fabiomachadofonseca** [reply]
Ok, thanks for your response. I asked you because I found in the documentation (https://tfel.sourceforge.net/documents/mtest/mtest.pdf - page 4 -> ref [Milliard 14]) where someone has done it.

Nevertheless, I will give it a try ;)

Regards,
Fabio

**Reply by thelfer** [reply]
@fabiomachadofonseca That was more or less a decade ago :) I was not in charge of this internship, but I remember that it worked well. Let me know.
